homeassistant/components/palazzetti/manifest.json

```diff
@@ -3,8 +3,17 @@
   "name": "Palazzetti",
   "codeowners": ["@dotvav"],
   "config_flow": true,
+  "dhcp": [
+    {
+      "hostname": "connbox*",
+      "macaddress": "40F3857*"
+    },
+    {
+      "registered_devices": true
+    }
+  ],
   "documentation": "https://www.home-assistant.io/integrations/palazzetti",
   "integration_type": "device",
   "iot_class": "local_polling",
-  "requirements": ["pypalazzetti==0.1.6"]
+  "requirements": ["pypalazzetti==0.1.10"]
```

Comment:
since this is not really related to this PR, please bump in a separate PR

Reply:
Yes, it is related to the PR. The class PalazzettiClient is not returning the proper `host` in version 0.1.6, which prevents the discovery_confirm from showing it to the user.

Reply:
Even if it is related, it should still be done in a preliminary PR:
- one PR for the dependency bump
- one later PR for the new features

Reply:
Got it. I have created a separate [PR #129832](https://github.com/home-assistant/core/pull/129832) for the dependency bump. I'll merge and mark this one ready when the bump is merged.